epicsdbbuilder/__init__.py

```diff
@@ -17,4 +17,4 @@ def ExportModules(globs, base_module, modulenames):
 
 __all__ = ExportModules(globals(),
     'epicsdbbuilder',
-    ['dbd', 'recordbase', 'fanout', 'recordset', 'recordnames', 'parameter'])
+    ['dbd', 'recordbase', 'fanout', 'recordset', 'recordnames', 'parameter', 'const_array'])
```

Comment:
Given the fact this is __init__.py rather then a regular file, would you mind if I add line breaks leaving a single name per line and order it alphabetically? I think it can make it more readable and I need to change the line regardless the style.